packages/terra-section-header/tests/jest/SectionHeader.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import SectionHeader from '../../src/SectionHeader';
+
+describe('SectionHeader', () => {
+  const defaultRender = <SectionHeader title="foo" />;
+
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render without an accordion icon when no \'onClick()\' is passed', () => {
+    const sectionHeader = <SectionHeader title="foo" />;
+    const wrapper = render(sectionHeader);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render with an accordion icon when an \'onClick()\' callback is passed', () => {
+    const sectionHeader = <SectionHeader title="foo" onClick={() => {}} />;
+    const wrapper = render(sectionHeader);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render with an accordion icon in the open position when an \'onClick()\' callback and \'isOpen\' is passed', () => {
+    const sectionHeader = <SectionHeader title="foo" onClick={() => {}} isOpen />;
+    const wrapper = render(sectionHeader);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class section-header', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper.prop('className')).toContain('section-header');
+  });
+
+  it('should have the class is-interactable when \'onClick()\' is provided.', () => {
+    const sectionHeader = <SectionHeader title="foo" onClick={() => {}} />;
+    const wrapper = shallow(sectionHeader);
+    expect(wrapper.type()).toEqual('div');
+    expect(wrapper.hasClass('is-interactable')).toEqual(true);
```

Comment:
A snapshot test would be better here.

Reply:
In that case, it's being covered by a case above